### PR TITLE
Tj fetching

### PR DESCRIPTION
**Pull request by archeni**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Testing Instructions

Please describe the tests required to verify your changes. Provide instructions so PR Tester can check functionality. Please also list any relevant details for your tests

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings or errors
- [ ] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

File: scripts/weather/WeatherProvider.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  //The spread operator makes this quick work
+  return [...weatherCollection];
+}
+
+export const getWeather = (city) => {
+  return fetch(`https://api.openweathermap.org/data/2.5/forecast?q=${city}&appid=${settings.weatherKey}`)
+  .then(response => response.json())
+  .then(parsedResponse => {
+      weatherCollection = parsedResponse.list
+      console.log(parsedResponse.list)
```
Comment (JSchugt) [inline scripts/weather/WeatherProvider.js:17]: This can be removed

File: scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,11 @@
+import { getAttractions } from "./attractions/AttractionProvider.js";
+import { getEateries } from "./eateries/EateryProvider.js";
 import {sendDropDownToDom} from "./nav/dropDown.js";
+import { getParks } from "./parks/ParkProvider.js";
+import { getWeather } from "./weather/WeatherProvider.js";
 
+getParks()
```
Comment (JSchugt) [inline scripts/main.js:7]: These get functions should be called by the function that are writing to the DOM and not main.

File: scripts/eateries/EateryProvider.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  return [...eateriesCollection];
+}
+
+export const getEateries = () => {
+
+  return fetch("http://holidayroad.nss.team/eateries")
+  .then(response => response.json())
+  .then(parsedResponse => {
+      eateriesCollection = parsedResponse
+      console.log(parsedResponse)
```
Comment (JSchugt) [inline scripts/eateries/EateryProvider.js:16]: This can be removed

File: scripts/parks/ParkProvider.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  return [...parksCollection];
+}
+
+export const getParks = () => {
+
+  return fetch(`https://developer.nps.gov/api/v1/parks?api_key=${settings.npsKey}`)
+  .then(response => response.json())
+  .then(parsedResponse => {
+      parksCollection = parsedResponse.data
+      console.log(parsedResponse.data)
```
Comment (JSchugt) [inline scripts/parks/ParkProvider.js:18]: Also this console.log

**Review by JSchugt — COMMENTED** [review]
Some things to address

**Review by cculkin8 — APPROVED** [review]
Looks great to me! All error codes fixed

**Pull request merged**